README.md

```diff
@@ -0,0 +1,32 @@
+# JDT Debug
+
+JDT Debug implements Java debugging support and works with any JDPA-compliant target Java VM.
+It is implemented on top of the language independent "debug model" provided by the platform debugger. 
+JDT debug provides the following debugging features:
+
+- Launching of a Java VM in either run or debug mode
+- Attaching to a running Java VM
+- Expression evaluation in the context of a stack frame
+- Scrapbook pages for interactive Java code snippet evaluation
+- Dynamic class reloading where supported by Java virtual machine
```

Comment:
This section is mainly based on the content of
https://eclipse.dev/eclipse/debug/index.php